hackathon_site/dashboard/frontend/src/components/cart/CartSummary/CartSummary.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styles from "./CartSummary.module.scss";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import TitledPaper from "components/general/TitledPaper/TitledPaper";
+import { useDispatch } from "react-redux";
+import { submitOrder } from "slices/hardware/cartSlice";
+
+interface CartQuantity {
+    cartQuantity: number;
+}
+
+const CartSummary = ({ cartQuantity }: CartQuantity) => {
```

Comment:
Optional: change this to use the selector to get the cart quantity from the store instead of passing it as a prop. I know this isn't part of the ticket, so I'll leave it as optional.